web/html/xui/form/outbound.html

```diff
@@ -66,6 +66,31 @@
         </a-form-item>
 </template>
 
+<!-- wireguard settings -->
+<template v-if="outbound.protocol === Protocols.Wireguard">
+    <a-form-item label='{{ i18n "pages.xray.outbound.MTU" }}'>
+        <a-input-number v-model.number="outbound.settings.MTU"></a-input-number>
+    </a-form-item>
+    <a-form-item label='{{ i18n "pages.xray.outbound.DNS" }}'>
```

Comment:
Please don't use i18n translation for well-known abbreviations.
DNS and MTU should be used as same as english words.